### PR TITLE
feat: Introducing Analytics

### PR DESCRIPTION
**Pull request by douglasmuraoka**
Introduces analytics into the framework by tracking the command ran by the users. The request itself is done by a separate process, in order to avoid blocking the main process.

### CODE REVIEW

File: packages/ant-util-analytics/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@back4app/ant-util-analytics",
+  "version": "0.0.17",
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=10.10"
+  },
+  "preferGlobal": true,
```
Comment (davimacedo) [inline packages/ant-util-analytics/package.json]: This line should be removed. It means that you want it to be installed with npm install ant-util-analytics --global and I think that's not the case, right?

File: packages/ant-util-analytics/package.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "author": "Back4App Inc.",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/back4app/antframework/issues"
+  },
+  "homepage": "https://github.com/back4app/antframework/tree/master/packages/ant-util-analytics#readme",
+  "dependencies": {
+    "parse": "^2.1.0"
+  },
+  "devDependencies": {
```
Comment (davimacedo) [inline packages/ant-util-analytics/package.json]: It is missing jsdoc and minami.

File: packages/ant-util-analytics/package.json (hunk 1 above)
Comment (douglasmuraoka) [reply packages/ant-util-analytics/package.json]: Yes, makes no sense to install It globally :)

File: packages/ant-util-analytics/package.json (hunk 2 above)
Comment (douglasmuraoka) [reply packages/ant-util-analytics/package.json]: Done!

**Pull request merged**